argoverse/utils/centerline_utils.py

```diff
@@ -84,7 +84,7 @@ def centerline_to_polygon(
     add_cond1 = np.logical_and(dx < 0, dy < 0)
     add_cond2 = np.logical_and(dx < 0, dy > 0)
     subtract_cond = np.logical_or(subtract_cond1, subtract_cond2)
-    add_cond = np.logical_or(add_cond1, add_cond2)
+    np.logical_or(add_cond1, add_cond2)
```

Comment:
thanks for the updates. i think this can be removed, also

Reply:
👍